nuclia/tests/test_manage/test_account.py

```diff
@@ -0,0 +1,8 @@
+from nuclia.sdk.accounts import NucliaAccounts
+from nuclia.tests.fixtures import TESTING_ACCOUNT_SLUG
+
+
+def test_list_accounts(testing_config):
+    accounts = NucliaAccounts()
+    all = accounts.list()
```

Comment:
nit: I would use another variable name, just in case, as `all` is part of the python syntax

Reply:
right